site/content/contribute/developer-setup/docker.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Docker Services"
+heading: "Docker Services for Development"
+description: "Learn about using the various Docker services to support your development environment."
+weight: 1
+---
+
+By default, only a small number of required Docker services are started to support basic development:
+
+```
+ENABLED_DOCKER_SERVICES="postgres mysql inbucket"
+```
+
+But there are many additional services ready to work with your local environment. Note that some services will require a Mattermost Enterprise license.
+
+```
+ENABLED_DOCKER_SERVICES="postgres mysql inbucket minio openldap dejavu keycloak elasticsearch grafana prometheus promtail loki"
+```
+
+To customize which services are started, either export the above environment variable or copy [`config.mk`](https://github.com/mattermost/mattermost/blob/master/server/config.mk) as `config.override.mk` to tune appropriately.
+
+## postgres
+
+From https://www.postgresql.org/:
+
+> The official site for PostgreSQL, the world's most advanced open source database.
+
+This is the default and recommended database to use with Mattermost. No additional configuration should be required, but the following settings apply to a Mattermost instance using Postgres:
```

Comment:
Is the official name ``Postgres`` or ``PostgreSQL``?

Reply:
Postgres is the widely accepted "nickname" (and original name) of the project, and something you would expect to hear from developers (vs. enunciating all of "post-gres-q-l"), but I'm 0/5 on whether that more casual language should be used here.

Reply:
Thanks! I think we can leave this as-is.